app/User.php

```diff
@@ -15,7 +15,7 @@ class User extends Authenticatable
      * @var array
      */
     protected $fillable = [
-        'first_name', 'last_name', 'email', 'password',
+        'first_name', 'last_name', 'email', 'password', 'email_token',
```

Comment:
I'm not enough into Laravel here, but I don't think the email_token should be fillable.

Reply:
It has to be there because when it isn't there it can't generated the token in the create function (RegisterController)